htsget_server/database.py

```diff
@@ -6,7 +6,7 @@
 from config import DB_PATH, BUCKET_SIZE, HTSGET_URL
 
 
-engine = create_engine(DB_PATH, echo=False)
+engine = create_engine(DB_PATH, echo=True)
```

Comment:
This should be switched back for performance

Reply:
When I turned this back to False on my machine, `make test-integration` ran like a dream. So speedy!

Reply:
I pushed this change myself, since it was just sitting there in my git :)